.maintain/chaostest/src/hypervisor/deployment/deployment.js

```diff
@@ -0,0 +1,57 @@
+const k8s = require('./k8s')
+const CONFIG = require('../../CONFIG')
+const { pollUntil } = require('../utils/wait')
+
+const readOrCreateNamespace = async (namespace) => {
+    try {
+        console.log('Reading namespace')
+        await k8s.readNameSpace(namespace)  // if namespace is available, do not create here
+    } catch (error) {
+        console.log('Namespace not presented, creating...')
+        await k8s.createNameSpace(namespace)
+    }
+    CONFIG.namespace = namespace
```

Comment:
This is a global variable. Take the advice below with a grain of salt as I am not an experience JS developer:

Using global variables makes code difficult to comprehend, test and maintain. I think there are a lot of resources out there going into further details. Is there a way around it?

Why not have `readOrCreateNamespace` take a `Config` as a parameter?